### PR TITLE
[UI Test] Sales Order Added

### PR DESCRIPTION
**Pull request by vishdha**
Summary:
added multi currency, price_list, multi UOM, taxes and charges, item_wise tax, discount on Item,
Discount on Grand total, terms and conditions
anim

### CODE REVIEW

File: erpnext/selling/doctype/sales_order/test_sales_order.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+			// get tax account head details
+			assert.ok(cur_frm.doc.taxes[0].account_head=='CGST - '+frappe.get_abbr(frappe.defaults.get_default('Company')), " Account Head abbr correct");
+			// calculate_taxes_and_totals
+			assert.ok(cur_frm.doc.rounded_total== 4414.50, "Round total correct");
+		},
+		() => cur_frm.print_doc(),
+		() => frappe.timeout(1),
+		() => {
+			assert.ok($('.btn-print-print').is(':visible'), "Print Format Available");
+			assert.ok(RegExp(/SO-\d\d\d\d\d/g).test($("#header-html small").text()));
```
Comment (rmehta) [inline erpnext/selling/doctype/sales_order/test_sales_order.js]: why regex, save the name somewhere and check for the full name.

File: erpnext/tests/ui/make_fixtures.js [hunk 2]
```diff
@@ -205,6 +205,21 @@ $.extend(frappe.test_data, {
 			{title: "Test Term 2"}
 		]
 	},
+	"Sales Taxes and Charges Template": {
+		"TEST In State GST": [
+			{title: "TEST In State GST"},
+			{taxes:[
+				[
+					{charge_type:"On Net Total"},
+					{account_head:"CGST - "+frappe.get_abbr(frappe.defaults.get_default("Company")) }
```
Comment (rmehta) [inline erpnext/tests/ui/make_fixtures.js]: are these accounts created?

File: erpnext/selling/doctype/sales_order/test_sales_order.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+QUnit.module('Sales Order');
+
+QUnit.test("test sales order", function(assert) {
+	assert.expect(7);
+	let done = assert.async();
+	frappe.run_serially([
+		/* () => frappe.tests.setup_doctype('Customer'),
```
Comment (rmehta) [inline erpnext/selling/doctype/sales_order/test_sales_order.js]: remove comments

**Review by rmehta — CHANGES_REQUESTED** [review]

File: erpnext/tests/ui/make_fixtures.js (hunk 2 above)
Comment (vishdha) [inline erpnext/tests/ui/make_fixtures.js]: yes

**Pull request closed without merging**